pkg/provider/kubernetes/crd/traefikio/v1alpha1/tlsoption.go

```diff
@@ -44,7 +44,8 @@ type TLSOptionSpec struct {
 	// ALPNProtocols defines the list of supported application level protocols for the TLS handshake, in order of preference.
 	// More info: https://doc.traefik.io/traefik/v3.3/https/tls/#alpn-protocols
 	ALPNProtocols []string `json:"alpnProtocols,omitempty"`
-
+	// DisableSessionTickets when set to true, disables TLS session resumption via session tickets.
+	DisableSessionTickets bool `json:"sessionTicketsDisabled,omitempty"`
```

Comment:
I think you forgot to change the attribute name in the struct tag. Once the tag is renamed, remember to regenerate the CRDs with the `make generate-crd` command.

```suggestion
	DisableSessionTickets bool `json:"disableSessionTickets,omitempty"`
```